pkg/rpc/client.go

```diff
@@ -152,6 +155,29 @@ func (c *Client) SetClient(cli *http.Client) {
 	}
 }
 
+// CalculateInputs creates input transactions for the specified amount of given
+// asset belonging to specified address. This implementation uses GetUnspents
+// JSON-RPC call internally, so make sure your RPC server suppors that.
+func (c *Client) CalculateInputs(address string, asset util.Uint256, cost util.Fixed8) ([]transaction.Input, util.Fixed8, error) {
+	var utxos core.UnspentBalances
+
+	resp, err := c.GetUnspents(address)
+	if err != nil || resp.Error != nil {
```

Comment:
why GetUnspents just can't return err, ie having check for resp.Err inside there? 
and here leave only check for err != nil

Reply:
Yeah, but all the other client calls are currently following this convention. It's strange, I think it's wrong, but at this moment we're better doing things consistently and then we'll solve it for all calls.